pkg/volume/util/fsquota/project.go

```diff
@@ -345,7 +345,7 @@ func removeProjectID(path string, ID common.QuotaID) error {
 	if err == nil {
 		defer closeProjectFiles(fProjects, fProjid)
 		list := readProjectFiles(fProjects, fProjid)
-		writeProjid := true
+		var writeProjid bool
```

Comment:
I think default value of bool is false

Reply:
@anderseknert that's correct, but the default value doesn't have any affect here as the `writeProjid` value is being reset on the next line anyway.